env0/resource_cost_credentials.go

```diff
@@ -2,7 +2,6 @@ package env0
 
```

Comment:
This code has been refactored. Easier to support the Update use-case without duplicating code.

Reply:
might be easier to just review the whole file.